velox/common/base/Counters.cpp

```diff
@@ -219,9 +219,9 @@ void registerVeloxMetrics() {
   // Total number of SSD evict log file open errors.
   DEFINE_METRIC(kMetricSsdCacheOpenLogErrors, facebook::velox::StatType::SUM);
 
-  // Total number of errors while deleting SSD checkpoint files.
+  // Total number of errors while deleting SSD checkpoint/evictlog files.
   DEFINE_METRIC(
-      kMetricSsdCacheDeleteCheckpointErrors, facebook::velox::StatType::SUM);
+      kMetricSsdCacheDeleteFileErrors, facebook::velox::StatType::SUM);
```

Comment:
the naming seems confusing, since the delete file errors only include two types and doesn't have SSD file?